src/test/ui/const-generics/array-impls/alloc-traits-impls-length-32.rs

```diff
@@ -18,6 +18,10 @@ pub fn yes_array_into_vec<T>() -> Vec<T> {
     [].into()
 }
 
+pub fn yes_array_into_box<T>() -> Box<[T]> {
+    [].into()
+}
```

Comment:
There should also be a `no_array_into_box` in `alloc-types-no-impls-length-33.rs`.

Reply:
I tried doing this but it failed, not sure how to fix it. What is the proper way to do that?

Reply:
There are examples in that file. I'm not sure how more to help without knowing the error message.

Reply:
I sent the error message in a comment.